src/anomalib/data/base/datamodule.py

```diff
@@ -227,3 +227,41 @@ def test_dataloader(self) -> EVAL_DATALOADERS:
     def predict_dataloader(self) -> EVAL_DATALOADERS:
         """Use the test dataloader for inference unless overridden."""
         return self.test_dataloader()
+
+    @property
+    def user_transform(self) -> Transform:
```

Comment:
Not sure about the naming, but I don't have a better suggestion, though. How about `custom_transform`?

Reply:
Yeah I agree but wasn't sure what would be a better name. Maybe just `transform`?

Reply:
if it does not conflict with any others, I would prefer `transform`